packages/venia-concept/index.module.css

```diff
@@ -6,6 +6,27 @@
     font-family: var(--drexel-global-fontFamily-serif);
 }
 
+:global(.footer-socialLinks) {
+    display: flex;
+    flex: 1;
+}
+
+:global(.footer-socialLinks li) {
+    margin-right: 10px;
+}
+
+:global(.footer-socialLinks .footer-socialIcon){
+    height: 12px;
+    width: 15px;
+}
+
+:global(.footer-socialLinks .footer-socialIcon:hover){
+    color: rgb(var(--venia-global-color-ligthsteelblue));
+    font-size: 2em;
+    transition: all 0.2s ease-in-out;
+}
+
+
```

Comment:
I prefer using a component approach in work with styles.
So, it's better to move them to the footer.module.css  and import it in venia-concept